seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonToRowConverters.java

```diff
@@ -70,7 +70,9 @@ public class JsonToRowConverters implements Serializable {
                     .toFormatter();
 
     public static final String FORMAT = "Common";
-
+    private static final String CURRENT_DATE = "CURRENT_DATE";
+    private static final String CURRENT_TIME = "CURRENT_TIME";
+    private static final String CURRENT_TIMESTAMP = "CURRENT_TIMESTAMP";
```

Comment:
It's weird to put this into JsonToRowConverters. Why not just put it into fake source connector?

Reply:
I also discovered it and changing it.